ffi/capi/src/bidi.rs

```diff
@@ -22,6 +22,7 @@ pub mod ffi {
     #[diplomat::opaque]
     /// An ICU4X Bidi object, containing loaded bidi data
     #[diplomat::rust_link(icu::properties::props::BidiClass, Struct)]
+    #[diplomat::attr(demo_gen, disable)]
```

Comment:
Issue: Why disable?

If a type already doesn't have a signature compatible with demo_gen, we don't need to explicitly disable anything.

Please write comments at least.

Reply:
https://github.com/rust-diplomat/diplomat/pull/867 made a lot more methods "compatible", however many of them are not useful to expose in the demo. This PR does not disable any methods that were previously generated